src/core/inlines.js

```diff
@@ -22,6 +22,82 @@ import { renderInlineCitation } from "./render-biblio";
 export const name = "core/inlines";
 export const rfc2119Usage = {};
 
+/**
+ * @param {string} matched
+ * @param {DocumentFragment} df
+ */
+function inlineRFC2119Matches(matched, df) {
+  matched = matched.split(/\s+/).join(" ");
+  df.appendChild(
+    hyperHTML`<em class="rfc2119" title="${matched}">${matched}</em>`
+  );
+  // remember which ones were used
+  rfc2119Usage[matched] = true;
+}
+
+/**
+ * @param {string} matched
+ * @param {DocumentFragment} df
+ */
+function inlineXrefMatches(matched, df) {
+  const ref = matched
+    .replace(/^\{{3}/, "")
+    .replace(/\}{3}$/, "")
+    .trim();
+  if (ref.startsWith("\\")) {
+    df.appendChild(document.createTextNode(`{{{${ref.replace(/^\\/, "")}}}}`));
+  } else {
+    df.appendChild(idlStringToHtml(ref));
+  }
+}
+
+/**
+ * @param {string} matched
+ * @param {DocumentFragment} df
+ * @param {Text} txt
+ * @param {Object} conf
+ */
+function inlineBibrefMatches(matched, df, txt, conf) {
+  let ref = matched;
+  ref = ref.replace(/^\[\[/, "");
+  ref = ref.replace(/\]\]$/, "");
```

Comment:
```js
const ref = matched.slice(2, -2);
```

Reply:
@himanish-star, if you adopt these suggestions, please be sure you document what `.slice()` is chopping off. The current code is really obvious/clear - but `.slice(2, -2)` won't be.

Reply:
i.e., "// chops off "[[" at start, and "]]" at end"

Reply:
Currently it's somewhat unclear what's the assumptions here (because the monolithic regex) so a proper comment will be great 👍🚀

Reply:
fixing this and all the above suggestions in a while :)

Reply:
> proper comment will indeed be great :+1: :rocket: 

will do so

Reply:
done